### PR TITLE
Finish converting link-to and input to angle brackets

### PR DESCRIPTION
**Pull request by jenweber**
Also added labels to form inputs, partially addresses #798

### CODE REVIEW

File: guides/release/templates/input-helpers.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
-The [`{{input}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=input)
-and [`{{textarea}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=textarea)
+The [`Input`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=input)
+and [`textArea`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=textarea)
```
Comment (rwjblue) [inline guides/release/templates/input-helpers.md]:
```suggestion
and [`Textarea`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=textarea)
```

https://github.com/emberjs/rfcs/blob/master/text/0459-angle-bracket-built-in-components.md#textarea

**Comment by jenweber** [comment]
I think I need to put some `@` in front of these - will add another commit after I do some testing.

File: guides/release/templates/input-helpers.md [hunk 2]
```diff
@@ -48,7 +51,8 @@ current context.
 To dispatch an action on specific events such as `key-press`, use the following
 
 ```handlebars
-{{input value=this.firstName key-press=(action "updateFirstName")}}
+<label for="firstname">First Name</label>
+<Input value={{this.firstName}} key-press={{action "updateFirstName" id="firstname"}} />
```
Comment (rwjblue) [inline guides/release/templates/input-helpers.md]:
```suggestion
<Input @value={{this.firstName}} @key-press={{action "updateFirstName" id="firstname"}} />
```

File: guides/release/templates/input-helpers.md [hunk 3]
```diff
@@ -67,11 +71,12 @@ More [events types](https://www.emberjs.com/api/ember/release/classes/Component)
...
 
 You can also use the
-[`{{input}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=input)
+[`Input`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=input)
 helper to create a checkbox by setting its `type`:
 
 ```handlebars
-{{input type="checkbox" name="isAdmin" checked=this.isAdmin}}
+<label for="is-admin">Admin: </label>
+<Input type="checkbox" name="isAdmin" checked={{this.isAdmin}} id="is-admin"/>
```
Comment (rwjblue) [inline guides/release/templates/input-helpers.md]:
```suggestion
<Input @type="checkbox" @name="isAdmin" @checked={{this.isAdmin}} id="is-admin"/>
```

File: guides/release/templates/input-helpers.md [hunk 4]
```diff
@@ -91,19 +96,21 @@ Which can be bound or set as described in the previous section.
 Checkboxes are a special input type. If you want to dispatch an action on a certain [event](https://www.emberjs.com/api/ember/release/classes/Component), you will always need to define the event name in camelCase format:
 
 ```handlebars
-{{input type="checkbox" keyPress=(action "updateName")}}
+<label for="firstname">First Name</label>
+<Input type="checkbox" key-press={{action "updateName"}} id="firstname" />
```
Comment (rwjblue) [inline guides/release/templates/input-helpers.md]:
```suggestion
<Input @type="checkbox" @key-press={{action "updateName"}} id="firstname" />
```

File: guides/release/templates/input-helpers.md [hunk 5]
```diff
@@ -91,19 +96,21 @@ Which can be bound or set as described in the previous section.
...
 ```
 
 
 ## Text Areas
 
 ```handlebars
-{{textarea value=this.name cols="80" rows="6"}}
+<label for="firstname">First Name</label>
+<Textarea value={{this.name}} cols="80" rows="6" id="firstname" />
```
Comment (rwjblue) [inline guides/release/templates/input-helpers.md]:
```suggestion
<Textarea @value={{this.name}} cols="80" rows="6" id="firstname" />
```

File: guides/release/templates/input-helpers.md [hunk 6]
```diff
@@ -128,7 +135,8 @@ Will bind the value of the text area to `name` on the current context.
 You might need to bind a property dynamically to an input if you're building a flexible form, for example. To achieve this you need to use the [`{{get}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/get?anchor=get) and [`{{mut}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/mut?anchor=mut) in conjunction like shown in the following example:
 
 ```handlebars
-{{input value=(mut (get this.person this.field))}}
+<label for="firstname">First Name</label>
+<Input value={{mut (get this.person this.field)}} id="firstname" />
```
Comment (rwjblue) [inline guides/release/templates/input-helpers.md]:
```suggestion
<Input @value={{mut (get this.person this.field)}} id="firstname" />
```

**Comment by rwjblue** [comment]
@jenweber - I think I got them 😸

**Pull request merged**